src/tiles/tile-dispatcher.ts

```diff
@@ -154,6 +154,7 @@ export class TileDispatcher
               .currentHeight=${this.currentHeight}
               .collectionNameCache=${this.collectionNameCache}
               .baseImageUrl=${this.baseImageUrl}
+              .sortParam=${sortParam}
```

Comment:
Q: when does this every return null?

Reply:
So if you don't have any ?sort=xxx param in URL, it does return null value.